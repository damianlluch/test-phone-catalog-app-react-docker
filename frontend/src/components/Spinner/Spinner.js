import React from 'react';
import { RingLoader } from 'react-spinners';
import './Spinner.css';

const Spinner = () => (
  <div className="Spinner">
    <RingLoader color="lightblue" loading />
  </div>
);

export default Spinner;
