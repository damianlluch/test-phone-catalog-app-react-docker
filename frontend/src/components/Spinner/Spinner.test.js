import React from 'react';
import renderer from 'react-test-renderer';
import Spinner from './Spinner';

describe('<Spinner/>', () => {
  it('renders by default', () => {
    const tree = renderer.create(<Spinner />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
