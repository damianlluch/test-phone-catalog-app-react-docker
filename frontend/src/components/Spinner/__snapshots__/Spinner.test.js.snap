// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner/> renders by default 1`] = `
<div
  className="Spinner"
>
  <div
    className="css-jxiqlq"
  >
    <div
      className="css-b2ebt3"
    />
    <div
      className="css-1j42v1p"
    />
  </div>
</div>
`;
